packages/shared/lib/utils/transactionObject.ts

```diff
@@ -0,0 +1,33 @@
+import { Payload } from '@lib/typings/message'
```

Comment:
If you're creating a new utils module, let's go ahead and put it in `core`, so `core/utils/transaction-object.ts` along with a barrel export file for it. Anything in these folders should use the `kebab-case` naming convention for filenames.

Arguably this belongs in a `wallet` module within `common` (esp. since you're importing a lot of functions from `wallet.ts`). I've mentioned to David how it'd be helpful having `wallet.ts` split up specific modules, e.g.
- `account` - retrieves account data, syncs accounts, handles addresses, handles transfers, account name, etc.
- `network` - handles anything network related like setting the client options, getting node info
- `stronghold` - handles backups, any software security-related things like mnemonic creation and verification, locking Stronghold, etc.
- `ledger` - handles specific Ledger-related stuff (not much really besides querying for the device status)
- `participation` - handles everything for voting and staking
- `migration` - handles everything for migration
- `storage` - handles storage stuff, i.e. managing PINs, encrypting / decrypting the storage, deleting storage

... and I think this util function would belong somewhere in there. We agreed to start doing this after the new bindings are up to speed.

Reply:
I can move it and follow the conventions, my only concern is its not really a library function per say for managing an object or context of the wallet. just a helper function so not sure it belongs in core?

Reply:
Yeah that's true, would be another good one for a library of specific UI functions.